packages/extension-vscode/README.md

```diff
@@ -0,0 +1,27 @@
+# webhint extension for Visual Studio Code
+
+Use `webhint` to improve your website - during development.
+
+This extension runs and reports diagnostics for workspace files based on
+`webhint` analysis.
+
+## Prerequisites
+
+This extension requires a local or global install of `webhint` to run.
+
+Run `npm install hint --save-dev` to add `webhint` to your project.
+Or run `npm install -g hint` to install `webhint` globally.
+
+## Configuration
+
+This extension uses your local `.hintrc` file to configure `webhint`.
+If no `.hintrc` file is found it defaults to `@hint/configuration-development`.
```

Comment:
Maybe we should link to the `configuration-development` index file?

Reply:
As in here: https://www.npmjs.com/package/@hint/configuration-development?

Or somewhere on webhint.io?

Reply:
As in: https://github.com/webhintio/hint/blob/master/packages/configuration-development/index.json